pkg/kubelet/kubeclientmanager/kubeclient_manager.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2020 Authors of Arktos.
+Copyright 2020 Authors of Arktos - file modified.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubeclientmanager
+
+import (
+	"k8s.io/api/core/v1"
+	clientset "k8s.io/client-go/kubernetes"
```

Comment:
minor: re-order the imports, the golang embedded imports first, then those of k8s.

Reply:
hmmm surprised "make update" didn't catch this. 

will do.

Reply:
fixed

Reply:
Oh I see, this order is enforced by GoLand